static/js/portico/header.js

```diff
@@ -13,4 +13,10 @@ $(function () {
             $(".dropdown").removeClass("show");
         }
     });
+
+    if ($(".desktop-redirect-links").length > 0) {
+        setTimeout(() => {
+            document.getElementById('desktop-redirect-link').click();
+        }, 2000);
+    }
```

Comment:
There’s no need to use JavaScript for this. Just add a `<meta http="Refresh">` tag to the template instead:

```jinja2
{% block customhead %}
<meta http-equiv="Refresh" content="2; {{ desktop_url }}">
{% endblock %}
```

Reply:
Great. Will change this. Thanks.